applications/chaineducation.md

```diff
@@ -68,12 +79,13 @@ Level 2: Up to $30,000, 3 approvals
 | Number | Deliverable | Specification | 
 | ------------- | ------------- | ------------- |
 | 0a. | Apache License 2.0 | All code will be published under Apache 2.0 |
-| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) deploy our smart contracts and send test transactions, which will show how the new functionality works. |
 | 0c. | Testing and it's Guide | Core functions will be fully covered by comprehensive unit tests written in Jest to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
 | 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
 | 0e. | Articles | We will publish a series of tutorials along with online workshops within our discord community so that we can educate our users the use of this platform. |
-| 0. | Creation of smart contracts using Ink. | These smart contracts will be responsible to distribute tokens to user once they unlock and achievements. Also, they will be responsible to mint achievement, milestone and course completion NFTs. IPFS will store key attributes of user profile. We will be using **erc20** token: https://github.com/paritytech/ink/blob/master/examples/erc20/lib.rs and **erc721** token to mint smart contract: https://github.com/paritytech/ink/blob/master/examples/erc721/lib.rs. One more smart contract would be required to send achievments NFT who complete the course. |
-| 1. | Oracle | Develop an oracle to fetch token price. |
+| 0. | Creation of smart contracts using Ink. | These smart contracts will be responsible to distribute tokens to user once they unlock and achievements. Also, they will be responsible to mint achievement, milestone and course completion NFTs. IPFS will store key attributes of user profile. We will be using **erc20** token: https://github.com/paritytech/ink/blob/master/examples/erc20/lib.rs and **erc721** token to mint smart contract: https://github.com/paritytech/ink/blob/master/examples/erc721/lib.rs. |
+| 1. | Creation of smart contracts using Ink. | Smart contract would be required to send achievments NFT who complete the course. This action would be triggered from the backend once user hits any milestone or completes her course. |
+| 2. | The DOT token | We will be using **DOT** token which is native to Polkadot platform to drive the platform so that users can buy the course and tokens will directly go to the creator's account. Creators can earn using this platform. |
```

Comment:
This is a list of deliverables. What is being delivered here? Also, which chain are you planning to deploy on? Polkadot doesn't support smart contracts.

Reply:
You are correct here. Polkadot doesn't support smart contracts. We will be using parachain to develop it. Then we will use relay chain to connect parachain with the main polkadot network.

Reply:
We will be deploying these above two smart contracts. Third smart contract needs to be coded from the scratch. @muskanmarwah1 is updating the application.